docs/README.md

```diff
@@ -97,8 +97,8 @@ These histories are used by offchain code to query state at particular blocks, a
 ##### Hooking Into EigenLayer Core
 
 The main thing that links an AVS to the EigenLayer core contracts is that when EigenLayer Operators register/deregister with an AVS, the AVS calls these functions in EigenLayer core:
-* [`DelegationManager.registerOperatorToAVS`][core-registerToAVS]
-* [`DelegationManager.deregisterOperatorFromAVS`][core-deregisterFromAVS]
+* [`AVSRegistry.registerOperatorToAVS`][core-registerToAVS]
+* [`AVSRegistry.deregisterOperatorFromAVS`][core-deregisterFromAVS]
```

Comment:
AVSDirectory

Reply:
Also need to update the link at the top of the file -- but currently we don't have docs for this contract/those methods. Lemme work on this today so I can get you a link...

Reply:
I've got these docs on `m2-mainnet-fixes` now: https://github.com/Layr-Labs/eigenlayer-contracts/blob/m2-mainnet-fixes/docs/core/AVSDirectory.md